### DISCUSSION TITLE
Impossible to POST an order -API

### DISCUSSION THREAD

**Discussion by mmedjane** [PrestaShop/PrestaShop#33505] [Q&A] [answered]
Hello everyone, 

I absolutely don't know what I'm doing wrong (especially since I was doing right before). Here the problem : I want to create an order, so I created a cart and then tried to do a POST request with this code : 
![post-order](https://github.com/PrestaShop/PrestaShop/assets/131868693/d736aa88-0e7e-4879-a9da-30313b4cc062)

 It used to work and then now I just get a "**Fatal error"** message, with no further explanation ! Even the console is not helpful at all 
![post order 2](https://github.com/PrestaShop/PrestaShop/assets/131868693/3128c38e-8f89-4a66-af98-86c1464550d2)

Has anyone an insight on what is happening ? 
Thanks in advance :)

**Comment by kpodemski** [comment, marked as answer]
Hi @mmedjane 

So, what has changed since you've done the previous requests, and everything has worked correctly? Did you install any third-party modules? Changed PS version? PHP version? It shouldn't fail after some time "just like that"
